functions_lab.js

```diff
@@ -0,0 +1,418 @@
+//
+// Output: `17`
+
+console.log('=========answer#3=======')
+
+
+function arraySum (arr) {
+
+  return
+
+  i++
+}
+
+let arr = [13,2,4,-2];
+
+
+const arraySum = (arr) => {
+  let sum = 0;
+for (let i = 0; i < arr.length; i++ ) {
+  sum += arr[i]
+  }
+
+  return sum
+}
+
+
+console.log(arraySum(arr))
+
+//
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+
+console.log('===========answer#4========')
+
+function sumOfTwoNums (num1, num2) {
+  return num1 + num2
+}
+let result1  = sumOfTwoNums(100, 2)
+console.log(result1)
+let result2 = sumOfTwoNums(14, 5)
+console.log(result2)
+
+
+//
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding
+//letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+//
+// console.log('=======answer#5=========')
+let num = 88
+const letterGrade = (num) => {
+  if (num >= 100) {
+    return "A+"
+  } else if (num <=99  && num >= 90){
+    return "A"
+  } else if (num >= 80 && num <= 89){
+    return "B"
+  } else if (num >= 70 && num <= 79) {
+    return "C"
+  } else if (num >= 65 && num <= 69){
+    return "D"
+  } else {
+    return "F"
+}
+}
+// let num = 88
+let result = letterGrade(num)
+
+
+console.log(result)
+
+
+// ## Question 6
+//
+// Implement a function named `repeatLog` that takes a string and a number
+//as parameters. The function should log `message` `count` number of times.
+//
+// Example:
+//
+// Input: `"+", 10`
+//
+// Output: `++++++++++`
+
+console.log('==========answer#6=======')
+// //
+function repeatLog (param1) {
+  return x = x.repeat(param1)
+  }
+let x = "+"
+let result = x.repeat(10)
+
+  console.log (result)
+//
+//
+// ## Question 7
+//
+// Write a function named `first` that takes a number and returns an array with
+// all the numbers from 1 to n
+//
+//
+// Example:
+//
+// Input: `first(3)`
+//
+// Output: `[1, 2, 3]`
+
+console.log('=========answer#7===========')
+//
+// const first = (num) => {
+//   return num[num.length]
+//   }
+//
+// let num = [3]
+// num.unshift(1, 2);
+//
+// console.log(num)
+//
+//
+// ## Question 8
+//
+// Write a function that logs the numbers from 1 to x, except:
+//
+// If the number if a multiple of 3, log `"Fizz"` instead of the number
+// If the number is a multiple of 5, log `"Buzz"` instead of the number
+// If the number is a multiple of 3 AND 5, log `"FizzBuzz"` instead of the number
+// Your function should take in one parameter: x (the number to count up to)
+//
+console.log('=========answer#8=========')
+
+function multiples (num) {
+  let num2 = []
+for (let i = 1; i <= num; i++) {
+  if (i % 3 === 0 && i % 5 === 0) {
+    num2.push("FizzBuzz")
```

Comment:
When the question asks for logging stuff, just console log them. No need to return anything from the function.